schema.prisma

```diff
@@ -29,6 +29,14 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+model BootcampNeli {
+  id                 String              @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
+  firstName          String              @map("first_name") @db.VarChar(100)
+  lastName           String              @map("last_name") @db.VarChar(100)
+  email              String              @unique @db.Citext
+   @@map("bootcamp_Neli")
```

Comment:
The db tables and columns should always be lowercase

I understand here it's a bit different because of the personal name, but that's the general rule